op-node/rollup/finality/finalizer_test.go

```diff
@@ -224,8 +275,53 @@ func TestEngineQueue_Finalize(t *testing.T) {
 		l1F.ExpectL1BlockRefByNumber(refD.Number, refD, nil) // to check finality signal
 		l1F.ExpectL1BlockRefByNumber(refD.Number, refD, nil) // to check what was derived from (same in this case)
 
+		l2F := &testutils.MockL2Client{}
+		defer l2F.AssertExpectations(t)
+		l2F.ExpectL2BlockRefByNumber(refA1.Number, refA1, nil)
```

Comment:
let's avoid repeating ourselves. you can extract it to be helper function that takes variable length number of ref blocks:

```
l2F = helperFunc(refA1, refB0, .... refC1)
```

Reply:
`helperSdkClientQueryBlockRangeBabylonFinalized`